src/components/pagination/_pagination.scss

```diff
@@ -196,7 +196,7 @@ $css--helpers: true;
 }
 
 @mixin pagination--x {
-  .#{$prefix}--data-table-v2-container + .#{$prefix}--pagination {
+  .#{$prefix}--data-table-v2-container .#{$prefix}--pagination {
```

Comment:
Is pagination now nested in a data table container? I feel like we should keep them as adjacent since this would be a big change